USER_MANUAL.md

```diff
@@ -42,16 +42,36 @@ Actors used in gyms are in `Contetnt\Actors`: add any new Actors to this directo
 ##### Empty gym
 * The template for creating new gyms. Copy this to use as a starting point for your own gym.
 
+##### Dynamic Components gym
+* Demonstrates that:
+  * Dynamic component are correctly added to and removed from a replicated Actor (with load-balancing enabled).
+* Contains:
+  * A character with a PlayerController with key bindings for adding and removing a component.
+* To setup the gym:
+  * Adjust the settings in "SpatialOS Settings -> Editor Settings -> Launch -> Launch configuration file options -> Server Workers" to include 4 servers in a 2x2 grid.
+    * Set both "Rectangle grid column count" and "Rectangle grid row count" to 2
+    * Set "Instances to launch in editor" to 4
+* To test the gym:
+    * Move the character until the inspector shows that the EntityACL and other server simulated components are on different workers
+    * Press "R" to add the TestDynamicComponent to the character.
+    * Observe the TestDynamicComponent added to the character components list in the inspector (and on the EntityACL)
+    * Press "T" to remove the TestDynamicComponent from the character.
```

Comment:
Do you need to do this? As a reader this feels part of the testing process, but I am not sure what I am looking for after it's happened - what constitutes a correct removal? Is it just seeing it disappear locally?

Reply:
Component is removed from component list, also disappears from ComponentPresence list - will make more explicit